client.go

```diff
@@ -314,6 +314,12 @@ func connect(c *ovndb) (err error) {
 	notifier := ovnNotifier{c}
 	ovsdb.Register(notifier)
 
+	// The cache is re-initialized. This is because when reconnecting, deletions
+	// are not propagated by MonitorTables and we would risk the cache to conain
+	// elements that are not in the db anymore
+	if reconnecting {
```

Comment:
Minor comment here: I think this arg is not needed. We can simply initialize the cache for both cases. We can remove the initialization at L346.

Reply:
Yup, that was the other option I had in mind but I didn't like much the fact that New leaves the object half initialized.
I will change that